$MirOS: ports/mailnews/pine/patches/patch-pine_pine_c,v 1.1 2005/05/04 23:37:41 tg Exp $
--- pine/pine.c.orig	Mon Sep 12 22:04:03 2005
+++ pine/pine.c	Fri Oct  7 19:12:55 2005
@@ -4430,7 +4430,7 @@ pine_mail_open(stream, mailbox, openflag
     static unsigned long streamcounter = 0;
 
     dprint(7, (debugfile,
-    "pine_mail_open: opening \"%s\"%s openflags=0x%x %s%s%s%s%s%s%s%s%s (%s)\n",
+    "pine_mail_open: opening \"%s\"%s openflags=0x%lx %s%s%s%s%s%s%s%s%s (%s)\n",
 	   mailbox ? mailbox : "(NULL)",
 	   stream ? "" : " (stream was NULL)",
 	   openflags,
@@ -4641,13 +4641,13 @@ pine_mail_open(stream, mailbox, openflag
 	       && !(flags & SP_USERFLDR)){
 		sp_set_ref_cnt(stream, sp_ref_cnt(stream)+1);
 		dprint(7, (debugfile,
-		       "pine_mail_open: permlocked: ref cnt up to %d\n",
+		       "pine_mail_open: permlocked: ref cnt up to %ld\n",
 		       sp_ref_cnt(stream)));
 	    }
 	    else if(sp_ref_cnt(stream) <= 0){
 		sp_set_ref_cnt(stream, 1);
 		dprint(7, (debugfile,
-		       "pine_mail_open: permexact: ref cnt set to %d\n",
+		       "pine_mail_open: permexact: ref cnt set to %ld\n",
 		       sp_ref_cnt(stream)));
 	    }
 
@@ -4703,13 +4703,13 @@ pine_mail_open(stream, mailbox, openflag
 	       && !(flags & SP_USERFLDR)){
 		sp_set_ref_cnt(stream, sp_ref_cnt(stream)+1);
 		dprint(7, (debugfile,
-		       "pine_mail_open: matched: ref cnt up to %d\n",
+		       "pine_mail_open: matched: ref cnt up to %ld\n",
 		       sp_ref_cnt(stream)));
 	    }
 	    else if(sp_ref_cnt(stream) <= 0){
 		sp_set_ref_cnt(stream, 1);
 		dprint(7, (debugfile,
-		       "pine_mail_open: exact: ref cnt set to %d\n",
+		       "pine_mail_open: exact: ref cnt set to %ld\n",
 		       sp_ref_cnt(stream)));
 	    }
 
@@ -4835,7 +4835,7 @@ pine_mail_open(stream, mailbox, openflag
 	     "pine_mail_open: asked for HALFOPEN so returning stream as is\n"));
 	    sp_set_ref_cnt(stream, sp_ref_cnt(stream)+1);
 	    dprint(7, (debugfile,
-		   "pine_mail_open: halfopen: ref cnt up to %d\n",
+		   "pine_mail_open: halfopen: ref cnt up to %ld\n",
 		   sp_ref_cnt(stream)));
 	    if(stream && stream->dtb && stream->dtb->name
 	       && !strcmp(stream->dtb->name, "imap")){
@@ -4942,7 +4942,7 @@ pine_mail_open(stream, mailbox, openflag
 
 	/* initialize the reference count */
 	sp_set_ref_cnt(retstream, 1);
-	dprint(7, (debugfile, "pine_mail_open: reset: ref cnt set to %d\n",
+	dprint(7, (debugfile, "pine_mail_open: reset: ref cnt set to %ld\n",
 		   sp_ref_cnt(retstream)));
 
 	if(sp_add(retstream, usepool) != 0 && usepool){
@@ -5946,7 +5946,7 @@ sp_flag(stream, flags)
     if(!stream)
       return;
 
-    dprint(9, (debugfile, "sp_flag(%s, 0x%x): %s%s%s%s%s%s%s%s\n",
+    dprint(9, (debugfile, "sp_flag(%s, 0x%lx): %s%s%s%s%s%s%s%s\n",
 	    (stream && stream->mailbox) ? stream->mailbox : "?",
 	    flags,
 	    flags ? "set" : "clear",
@@ -5973,7 +5973,7 @@ sp_unflag(stream, flags)
     if(!stream || !flags)
       return;
 
-    dprint(9, (debugfile, "sp_unflag(%s, 0x%x): unset%s%s%s%s%s%s\n",
+    dprint(9, (debugfile, "sp_unflag(%s, 0x%lx): unset%s%s%s%s%s%s\n",
 	    (stream && stream->mailbox) ? stream->mailbox : "?",
 	    flags,
 	    (flags & SP_LOCKED)     ? " SP_LOCKED" : "",
@@ -5986,7 +5986,7 @@ sp_unflag(stream, flags)
     sp_set_flags(stream, sp_flags(stream) & ~flags);
 
     flags = sp_flags(stream);
-    dprint(9, (debugfile, "sp_unflag(%s, 0x%x): result:%s%s%s%s%s%s\n",
+    dprint(9, (debugfile, "sp_unflag(%s, 0x%lx): result:%s%s%s%s%s%s\n",
 	    (stream && stream->mailbox) ? stream->mailbox : "?",
 	    flags,
 	    (flags & SP_LOCKED)     ? " SP_LOCKED" : "",
